File: js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { UnivecSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await UnivecSDK.test()
    equal(null !== testsdk, true)
  })

})

File: csharp/test/ExistsTest.cs
// Univec SDK exists test.

using Xunit;

using UnivecSdk;

namespace UnivecSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = UnivecSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}
